zerver/models.py

```diff
@@ -164,6 +163,13 @@ class Realm(models.Model):
     COMMUNITY = 2
     org_type = models.PositiveSmallIntegerField(default=CORPORATE)  # type: int
 
+    # This value is also being used in static/js/settings_bots.bots_allowed_for_non_admins_values.
+    # On updating it here, update it there as well.
+    ALL = 1
+    NO_GENERIC_BOTS = 2
+    NONE = 3
```

Comment:
Stylistically, I don't think these variables names are great, since `Realm.ALL` could refer to all sorts of possible things.

I think we probably want a set of value names more like this:
`BOT_CREATION_UNLIMITED`
`BOT_CREATION_LIMITED`
`BOT_CREATION_ADMINS_ONLY`

Reply:
We could do BOT_CREATION_EVERYONE to match what we call the option in the frontend. 
And maybe BOT_CREATION_LIMIT_GENERIC_BOTS for the second?

Reply:
Does it make sense to have a dict? Like Realm.bot_creation_permissions.LIMIT_GENERIC_BOTS